tests/patterns/python/scoped_wildcard.py

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
Very nice that you took the time to write tests to cover different cases, also the TODO ones.